kong.conf.default

```diff
@@ -185,6 +185,27 @@
                                   # `text/html`, `application/json`, and
                                   # `application/xml`.
 
+#client_max_body_size = 0        # Defines the maximum request body size allowed
```

Comment:
The Nginx default is `1m`. In the past, we've always respected Nginx's defaults when introducing new config variables that are just wrapping underplaying Nginx directives. Any reason for not doing so for this - sensitive - setting?

Additionally, when a config value is a 1-to-1 mapping of an Nginx directive, we also mention it in the description (controls the Nginx core module directive bearing the same name), and we include a note below with the link to the Nginx documentation (which is especially useful because it mentions size units like `k` and `m` that these directive use).